infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesEnvironmentProvisioner.java

```diff
@@ -51,6 +53,9 @@
 
   void provision(T k8sEnv, RuntimeIdentity identity) throws InfrastructureException;
```

Comment:
If I see correctly, this guys is now called only from `PluginBrokerManager`, where we do have available `KubernetesNamespace`. If so, I would just add namespace param to this method.